lib/routes/page.js

```diff
@@ -652,12 +652,22 @@ module.exports = function(crowi, app) {
    * @apiParam {String} revision_id
    */
   api.get = function(req, res){
-    var pagePath = req.query.path || null;
-    var pageId = req.query.page_id || null; // TODO: handling
-    var revisionId = req.query.revision_id || null;
+    const pagePath = req.query.path || null;
+    const pageId = req.query.page_id || null; // TODO: handling
+    const revisionId = req.query.revision_id || null;
 
-    Page.findPage(pagePath, req.user, revisionId)
-    .then(function(pageData) {
+    if (!pageId && !pagePath) {
```

Comment:
Question: Is there a possibility that `pageId` is 0 and `pagePath` is 0?

Reply:
Yes, it is possible beacuse this is an API that is called by any client.
If the client doesn't specify both `page_id` and `path`, `req.query` doesn't have enough parameter, so we need to validate them.

Reply:
Will you add verification at this PR?

Reply:
Yes, I will